src/aiogram_dialog/manager/message_manager.py

```diff
@@ -31,14 +31,24 @@
     ContentType.VOICE: "send_voice",
 }
 
+_INVALUD_QUERY_ID_MSG = (
+    "query is too old and response timeout expired or query id is invalid"
```

Comment:
tg sometimes changes error messages. one day I changed error message pattern like this and next day tg change it again =(

Reply:
![image](https://github.com/Tishka17/aiogram_dialog/assets/3669045/fc4364d6-1edf-436b-b27d-ea01e44b604d)
now ID is uppercase